src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/SetNoteStatusRequestHandlerTests.cs

```diff
@@ -190,5 +240,49 @@ public async Task ExternalUser_WithNoteUpdate_SaveChangesAsyncShouldBeCalled()
                 .Then(A.CallTo(() => context.SaveChangesAsync())
                     .MustHaveHappenedOnceExactly());
         }
+
+        [Fact]
+        public async Task ExternalUser_WithRejectedNoteUpdate_SaveChangesAsyncShouldBeCalled()
```

Comment:
again, that test is the same for 
ExternalUser_WithNoteUpdate_SaveChangesAsyncShouldBeCalled()
ExternalUser_WithRejectedNoteUpdate_SaveChangesAsyncShouldBeCalled()
ExternalUser_WithReturnedNoteUpdate_SaveChangesAsyncShouldBeCalled()

same tests, just different statuses, please use 1 method and pass 3 statuses as param

Reply:
done